plotRisk.py

```diff
@@ -21,11 +28,10 @@ class Character:
         def __init__(self, name):
                 self.name = name
 
-        def plotRisk(self, attacks): # goes through enemies potential attacks and damage according to our defense profile Me
+        def PlotRisk(self, attacks): # goes through enemies potential attacks and damage according to our defense profile Me
```

Comment:
A docstring and type hint would be nice here.

Reply:
Self implied